ecs-init/config/common.go

```diff
@@ -29,8 +31,18 @@ const (
 	AgentLogFile = "ecs-agent.log"
 
 	UnixSocketPrefix = "unix://"
+
+	// Default region name
```

Comment:
From https://blog.golang.org/godoc-documenting-go-code

> The convention is simple: to document a type, variable, constant, function, or even a package, write a regular comment directly preceding its declaration, with no intervening blank line. Godoc will then present that comment as text alongside the item it documents.   

`DefaultRegionName` is the ...